client/src/Components/Auth/styles.js

```diff
@@ -0,0 +1,37 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export default makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(4),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: theme.spacing(2),
+  },
+  root: {
```

Comment:
can you leave blank spaces between classes please? It makes it easier to read.

Also, can you consider moving these to a `css` file and then use the angular `ngClass` attribute to pick and choose which classes get used

Reply:
Ideally, use an `.scss` file. Look at the next comment on why I am saying that.